cfgov/cfgov/urls.py

```diff
@@ -210,8 +210,8 @@
 
 # TODO: Remove prototype landing page routes when all organisms and molecules have been implemented elsewhere.
 if settings.DEBUG :
-    urlpatterns.append(url(r'^sublanding-page-1/$', SheerTemplateView.as_view(template_name='sublanding-page-1/index.html'), name='sublanding-page-1'))
-    urlpatterns.append(url(r'^sublanding-page-2/$', SheerTemplateView.as_view(template_name='sublanding-page-2/index.html'), name='sublanding-page-2'))
+    urlpatterns.append(url(r'^landing-page/$', SheerTemplateView.as_view(template_name='landing-page/index.html'), name='landing-page'))
```

Comment:
This was deleted in #1154 which was merged when I was working on this. I'll be sure to remove. Good catch!